config/locales/cy.yml

```diff
@@ -42,8 +42,9 @@ cy:
 
       calculate:
         legend: Dewiswch sut mae eich cyflogwr yn cyfrannu
-        details: Mae eich cyflogwr yn penderfynu un ai i gyfrannu ar ran o’r cyflog neu’ch cyflog yn llawn. I gael gwybod pa un, bydd angen i chi wirio gyda’ch cyflogwr.
-
+        qualifying_earnings_title: Enillion cymhwyso
+        details_html: Gall eich cyflogwr ddewis os yw am wneud cyfraniadau ar ran o'ch cyflog (a elwir yn enillion cymhwyso<button type="button" class="popup-tip__button" data-dough-popup-trigger><span aria-hidden="true">i</span><span class="visually-hidden">sioe help</span></button>) neu ar eich cyflog llawn. I gael gwybod pa un, bydd angen i chi wirio gyda’ch cyflogwr.
```

Comment:
Is it possible to use the popup trigger partial in here? (It may not be, I don't know).